### PR TITLE
WIP - Az 239

### PR DESCRIPTION
**Pull request by takeit**
TODO (new request):

- this sidebar should list available subscriptions of type article, created in the paywall backend
- one subscription (with predefined price, period) should be connected with various articles
- at this moment one subscription can be connected with one article, the same subscription can't be connected to diffrent articles
- make use of Subscription Specification, where many specifications can be assigned to one subscription

Preview of the sidebar:

zrzut ekranu 2015-10-28 14 28 59

### CODE REVIEW

File: Controller/HookController.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    private function findOneOr404($articleNumber, $articleLanguage)
+    {
+        $em = $this->get('em');
+        $article = $em->getRepository('Newscoop\Entity\Article')->findOneBy(array(
+            'number' => $articleNumber,
+            'language' => $articleLanguage,
+        ));
+
+        if (!$article) {
+            throw new NotFoundHttpException('The article does not exist.');
```
Comment (m038) [inline Controller/HookController.php]: Should you use full namespace here or add it via use?

Comment (takeit) [inline Controller/HookController.php]: yeah, it has been removed mistakenly by cs fixer, added again

**Comment by m038** [comment]
:+1:

**Comment by takeit** [comment]
closed in favour of https://github.com/newscoop/plugin-NewscoopPaywallBundle/pull/28

**Pull request closed without merging**